quackling/llama_index/readers/docling_json_reader.py

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright IBM Corp. 2024 - 2024
+# SPDX-License-Identifier: MIT
+#
+
+import json
+from typing import Iterable
+
+from docling_core.types import Document as DLDocument
+from llama_index.core.schema import Document as LIDocument
+
+from quackling.llama_index.readers.base import BaseDoclingReader
+
+
+class DoclingJSONReader(BaseDoclingReader):
```

Comment:
I would also expose this as `quackling.llama_index.readers.DoclingJSONReader` (via `__init__.py`) and use like that in the test.